### PR TITLE
SPORTSHUB-147 under construction pages for metrics and gallerys

### PR DESCRIPTION
**Pull request by ReggieStar11**
## Why was your change needed?

Please detail here why you did the change you did.

## What was Changed?

Please detail here what were the main changes you made.

## Any extra notes for reviewers to know?

Write any extra notes you may have for the PR.
This may include specific code paths that need extra scrutiny, or how to review a specific portion of code.

### CODE REVIEW

File: frontend/app/(no-footer)/organiser/gallery/page.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import OrganiserNavbar from "@/components/organiser/OrganiserNavbar";
+import UnderConstruction from "@/components/UnderConstruction";
+
+const GalleryPage = () => {
+  return (
+    <div>
+      <div className="ml-14 mt-16">
+        <div className="max-w-5xl lg:mx-auto">
+          <OrganiserNavbar currPage="Gallery" />
```
Comment (owya490) [inline frontend/app/(no-footer)/organiser/gallery/page.tsx:9]: could we move this to layout.tsx as this is shared amongst all files?

File: frontend/public/images/coming_soon.jpeg
Comment (owya490) [inline frontend/public/images/coming_soon.jpeg:1]: if we aren't using this image, lets delete it!

**Review by owya490 — APPROVED** [review]
thanks for adding the pages looks good!

File: frontend/public/images/coming_soon.jpeg
Comment (ReggieStar11) [reply frontend/public/images/coming_soon.jpeg:1]: yep

**Review by owya490 — APPROVED** [review]
thanks for deleting photo

**Pull request merged**